dashboards/gemini-cli/gemini-cli-monitoring.json

```diff
@@ -134,76 +365,203 @@
                   },
                   "unitOverride": ""
                 }
+              },
+              {
+                "breakdowns": [],
+                "dimensions": [],
+                "legendTemplate": "total",
+                "measures": [],
+                "minAlignmentPeriod": "60s",
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "outputFullDuration": false,
+                  "timeSeriesFilter": {
+                    "aggregation": {
+                      "alignmentPeriod": "60s",
+                      "crossSeriesReducer": "REDUCE_SUM",
+                      "groupByFields": [],
+                      "perSeriesAligner": "ALIGN_DELTA"
+                    },
+                    "filter": "metric.type=\"custom.googleapis.com/gemini_cli/gemini_cli.token.usage\" metric.label.\"type\"!=\"tool\""
```

Comment:
FYI, the charting library is plotting this second dataset (total) on top of the first dataset (breakdown by type) as a stacked area chart:
https://screenshot.googleplex.com/9HkMRtMGbHAiz8u

Up to you if you want to keep it as it is or revert this. I don't think there is a way to do first dataset as a stacked area chart and total as a line chart on top unfortunately

Reply:
Removed it, thanks!